rwengine/src/script/ScriptTypes.cpp

```diff
@@ -95,3 +95,151 @@ template<> GameObject* ScriptArguments::getObject< VehicleObject >(unsigned int
 	auto it = getWorld()->objects.find( gameObjectID );
 	return (it == getWorld()->objects.end())? nullptr : it->second;
 }*/
+
+template<> float ScriptArguments::getParameter<float>(unsigned int arg) const
+{
+	return getParameters()[arg].realValue();
```

Comment:
We should be using at() here (or wrapping std::vector) instead of operator[] to get bounds-checking.